Analyzer/plugins/Analyzer.cc

```diff
@@ -7290,10 +7875,10 @@ void Analyzer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   desc.addUntracked("DeDxK",2.3)->setComment("K constant, really controlled by the config for each era");
   desc.addUntracked("DeDxC",3.17)->setComment("C constant, really controlled by the config for each era");
   desc.addUntracked("SaveTree",6)->setComment("0: do not save tree, 6: everything is saved");
-  desc.addUntracked<std::string>("DeDxTemplate","SUSYBSMAnalysis/HSCP/data/template_2017B.root")
+  desc.addUntracked<std::string>("DeDxTemplate","SUSYBSMAnalysis/HSCP/data/GiTemplate_EtaExtension.root")
```

Comment:
For which year and era this is for? Would be good to include in the name

Reply:
But the long term plan is to have this per era, right? also tagging @carolinecollard

Reply:
No. The plan is to get it per PU bins. as for EXO-18-002.

Reply:
so the template is for 2017+2018 up to 2.4 in eta :)

But teaser: in a next PR, new template will be provide with the saturation correction.